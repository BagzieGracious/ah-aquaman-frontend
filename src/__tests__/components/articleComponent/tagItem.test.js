import { shallow } from "enzyme";
import React from "react";
import TagComponent from "../../../components/article/articleComponent/tagComponent";

describe("tag tests", () => {
  const props = ["test", "test1"];
  it("should not regress", () => {
    const wrapper = shallow(<TagComponent {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
