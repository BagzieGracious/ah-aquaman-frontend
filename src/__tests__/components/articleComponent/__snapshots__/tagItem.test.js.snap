// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tag tests should not regress 1`] = `ShallowWrapper {}`;
